packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -40,10 +40,17 @@ const DEFAULT_SERIALIZATION_CONTEXT = {
   elementsScrollPositions: createElementsScrollPositions(),
 }
 
+const shadowDomCreatedCb = jasmine.createSpy()
+
+beforeEach(() => {
+  shadowDomCreatedCb.calls.reset()
+})
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we usually:
* declare the spy in the `describe` callback (with a `Spy` suffix)
* create it in the `beforeEach` callback